packages/kit/src/core/adapt/builder.js

```diff
@@ -189,6 +199,124 @@ export function create_builder({
 			return build_data.app_path;
 		},
 
+		getServerAssets() {
```

Comment:
I'm not 100% sure about the naming of this method — the other `getFoo` methods just return a directory name. The `generateFoo` methods are slightly inconsistent with each other since some write a file to disk while `generateManifest` returns the contents of a manifest module, but the prefix feels slightly more appropriate since it's doing some work rather than just concatenating strings

Relatedly (since it's a question of API design) I'm still unclear on why `rootErrorPage` needs to be handled separately — it almost feels like it should be something like this

```ts
generateAssetList(routes): string[]
```

though it would be worth thinking through how it will be used by other adapters, to figure out if that makes sense

Reply:
How about `generateServerAssetList`?